src/transformers/models/swin2sr/modeling_swin2sr.py

```diff
@@ -893,12 +888,12 @@ def pad_and_normalize(self, pixel_values):
 
     @add_start_docstrings_to_model_forward(SWIN2SR_INPUTS_DOCSTRING)
     @add_code_sample_docstrings(
-        processor_class=_FEAT_EXTRACTOR_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        processor_class="AutoImageProcessor",
+        checkpoint="caidas/swin2SR-classical-sr-x2-64",
```

Comment:
actually I only mean `_EXPECTED_OUTPUT_SHAPE` and we can keep `_FEAT_EXTRACTOR_FOR_DOC` and `_CHECKPOINT_FOR_DOC`.

`_CHECKPOINT_FOR_DOC` is used in pipeline testing